environment.go

```diff
@@ -115,9 +115,9 @@ func (b Builder) newEnvironment(ctx context.Context) (*environment, error) {
 	replaced := make(map[string]string)
 	for _, r := range b.Replacements {
 		log.Printf("[INFO] Replace %s => %s", r.Old.String(), r.New.String())
-		cmd := env.newGoModCommand("edit",
+		cmd := env.newGoModCommand(ctx, "edit",
 			"-replace", fmt.Sprintf("%s=%s", r.Old.Param(), r.New.Param()))
-		err := env.runCommand(ctx, cmd, 10*time.Second)
+		err := env.runCommand(ctx, cmd)
```

Comment:
Do we have a default timeout? I think these were a "bug fix" for a weird case where init/edit would hang too. Thinking about CLI use in this case.

Reply:
Timeouts are disabled for CLI by default after #15 was reported.